concepts/booleans/about.md

```diff
@@ -0,0 +1,77 @@
+# About
+
+A Boolean expression is a question asked of one or more values which can be
+answered with `true` or `false`. A true expression returns `1`; a false
```

Comment:
The C track tends to follow the line-break-on-sentence-end convention - could these be updated to follow that?

Reply:
Okay, so from what I can see at looking at some other docs, it's okay to have a very wide line (80+), just break after period, correct?